client.py

```diff
@@ -91,14 +92,18 @@ def send_model():
             datablock.init_new_image(image.shape, label)
             datablock.image_data[-1] = image
 
+
+
     datablock_dict = {"pi01": datablock}
 
     model = person_classifier.train(datablock_dict)
+
     model.save('./network.pth')
+    
     state_dict = open('./network.pth', 'rb').read()
+    
```

Comment:
do `python3 -m autopep8 --in-place --aggressive --aggressive --recursive *.py common utils` for consistent formatting..